hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/table/action/commit/BaseSparkCommitActionExecutor.java

```diff
@@ -292,8 +294,16 @@ protected void setCommitMetadata(HoodieWriteMetadata<HoodieData<WriteStatus>> re
   @Override
   protected void commit(HoodieWriteMetadata<HoodieData<WriteStatus>> result) {
     context.setJobStatus(this.getClass().getSimpleName(), "Commit write status collect: " + config.getTableName());
-    commit(result.getWriteStatuses(), result, result.getWriteStats().isPresent()
-        ? result.getWriteStats().get() : result.getWriteStatuses().map(WriteStatus::getStat).collectAsList());
+    if (!config.getBoolean(WRITE_IGNORE_FAILED) && result.getWriteStatuses().filter(status -> status.getErrors().size() > 0).count() > 0) {
+      WriteStatus writeStatus = result.getWriteStatuses().filter(status -> status.getErrors().size() > 0).collectAsList().get(0);
+      HashMap<HoodieKey, Throwable> errors = writeStatus.getErrors();
+      // get first one of error info to show error details
+      HoodieKey key = errors.keySet().iterator().next();
```

Comment:
The exception is too simple.
If get the exception, we should provide more logs to find out what happened.
For example, which error happened when writing to which files?

Reply:
OK，I would add more infomation for it，Thanks~